rct229/rules/section6/section6rule2.py

```diff
@@ -51,18 +51,23 @@ def get_calc_vals(self, context, data=None):
             space_u = context.user
 
             # get allowance
-            lighting_power_allowance_p = table_9_6_1_lookup(
-                space_p["lighting_space_type"]
-            )
+            if space_p["lighting_space_type"] in [
+                GUEST_ROOM,
+                DORMITORY_LIVING_QUARTERS,
+            ]:
+                lighting_power_allowance_p = table_9_6_1_lookup(
+                    space_p["lighting_space_type"]
+                )["lpd"]
+            elif space_p["lighting_space_type"] == DWELLING_UNIT:
+                lighting_power_allowance_p = 0.6
 
-            # sum of space_p lighting power density
-            space_lighting_power_per_area_p = sum(
-                interior_lighting["power_per_area"]
-                for interior_lighting in space_p["interior_lighting"]
+            space_lighting_power_per_area_p = pint_sum(
+                find_all("$..interior_lighting[*].power_per_area", space_p),
```

Comment:
Since _interior_lighitng_ is directly under the Space, you don't need the "$.." at the beginining of the JSON path.  The same is true for line 69 below.